pydoc/developer/TIPS.md

```diff
@@ -0,0 +1,39 @@
+.. _build_tips:
```

Comment:
Again, this label won't work natively in markdonw. Update to 
<pre><code>
```{eval-rst}  
.. _build_tips:
```
</code></pre>

or remove since they are not used.